manifests/v/VictorIX/BlenderLauncher/2.4.2/VictorIX.BlenderLauncher.installer.yaml

```
@@ -0,0 +1,21 @@
+# Created with WinGet Updater using komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: VictorIX.BlenderLauncher
+PackageVersion: 2.4.2
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: Blender Launcher.exe
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2025-03-03
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/Victor-IX/Blender-Launcher-V2/releases/download/v2.4.2/Blender_Launcher_v2.4.2_Windows_x64.zip
+  InstallerSha256: DF25665ED4070FC6C69E1836F8FF1665D69377E593ACA5EDAAAFDF60C528CBBD

```

```suggestion
  InstallerSha256: 9C6195FDBF07130F0A05419D0EE610F720A851F1D864628427F79DEBB87C5856
```

https://github.com/DuckDuckStudio/winget-tools/actions/runs/13624318907/job/38078845162#step:6:8